src/main/pages/che-6/portable-workspaces/factories_json_reference.adoc

```diff
@@ -235,9 +251,9 @@ Define a command for your workspace in `factory.workspace.workspaceConfig.comman
 }
 ----
 
-In this example, we have created a named command `replace` which replaces `***` with a string in project’s README.md.
+In the preceding example, a named command `replace` is created. The command replaces `***` with a string in project `README.md` file.
```

Comment:
"The command replaces `***` with a string in the `README.md` file of the project."

Also, can we name the replacement string? "A string" is vague, while it clearly refers to "userId984hfy6" in the example `sed` command.

It would also be more accurate to say that it replaces _all_ occurrences of `***` (based on the use of `g` in `sed`).